variables.tf

```diff
@@ -28,17 +27,95 @@ variable "fallback_public_client_enabled" {
   default     = false
 }
 
+variable "group_membership_claims" {
+  description = "A set of strings containing membership claims issued in a user or OAuth 2.0 access token that the app expects. Possible values are None, SecurityGroup, DirectoryRole, ApplicationGroup or All."
+  type        = set(string)
+  default     = ["None"]
+}
+
 variable "identifier_uris" {
   description = "A map of user-defined URIs that uniquely identify this application within its Microsoft Entra ID tenant, or within a verified custom domain if this application is multi-tenant."
   type        = map(string)
   default     = {}
 }
 
+variable "known_client_applications" {
+  description = "A set of object IDs of applications that are pre-authorized to access this application."
+  type        = set(string)
+  default     = []
+}
+
 variable "service_management_reference" {
   description = "Reference application context information from a service management database, e.g. ServiceNow."
   type        = string
 }
 
+variable "login_url" {
+  description = "The URL where the service provider redirects the user to Microsoft Entra ID to authenticate."
+  type        = string
+  default     = null
+}
+
+variable "notes" {
+  description = "User-specified notes relevant for the management of the application."
+  type        = string
+  default     = null
+}
+
+variable "optional_access_token_claims" {
+  description = <<-EOT
+  A list of optional access token claims to include in the access token. The object has the following structure:
+    `additional_properties` - List of additional properties of the claim. If a property exists in this list, it modifies the behaviour of the optional claim. 
+    Possible values are: cloud_displayname, dns_domain_and_sam_account_name, emit_as_roles, include_externally_authenticated_upn_without_hash, include_externally_authenticated_upn, max_size_limit, netbios_domain_and_sam_account_name, on_premise_security_identifier, sam_account_name, and use_guid.
+    `essential` - Whether the claim specified by the client is necessary to ensure a smooth authorization experience.
+    `name` - The name of the optional claim. For predifined optional claims see https://learn.microsoft.com/en-us/entra/identity-platform/optional-claims-reference
+    `source` - The source of the claim. If source is absent, the claim is a predefined optional claim. If source is user, the value of name is the extension property from the user object.
+  EOT
```

Comment:
These multi-line descriptions tend to look pretty bad in the Terraform Registry GUI 😅 Is it feasible to shorten this while still keeping it somewhat useful?